site/app/templates/grading/electronic/GradingPanelHeader.twig

```diff
@@ -1,16 +1,20 @@
 {% import _self as self %}
 <script>
-    let student_info = {% if isStudentInfoPanel %} true {% else %} false {% endif %};
-    let grade_inquiry_info = {% if isGradeInquiryPanel %} true {% else %} false {% endif %};
-    let discussion_browser = {% if isDiscussionPanel %} true {% else %} false {% endif %};
-    let peer_info = {% if isPeerPanel %} true {% else %} false {% endif %};
-    let is_notebook = {% if is_notebook %} true {% else %} false {% endif %};
+    const autograding_results = {{ not isPeerGrader or (isPeerGrader and isPeerAutograding) ? true : false }};
```

Comment:
Use 0 and 1's here as discussed in the meeting

Reply:
Also be careful about comparing (===) 0 to false

Reply:
let student_info = {% if isStudentInfoPanel %} true {% else %} false {% endif %};
    These still work, so I think we should just use these statement because its much clearer than why we are using 0 and 1's

Reply:
You are right, we should just revert back to if else because of 
1. Strong comparison we are doing across the site.
2. True or false is much clearer.